README.md

```diff
@@ -10,6 +10,10 @@ Frontend administration application for the digital marketplace.
 
 ## Quickstart
 
+System requirements include a working node.js and npm.
+
+- note that on macOS, a `brew install` of the latest node/npm results in `Error: Cannot find module 'internal/fs'` - use nodejs.org instead
```

Comment:
(Oh, I'm wrong, >=7.0 doesn't seem to work with npm either. I think? I'm confused anyway, and therefore clearly not the right person to write this comment. I still don't think it makes sense to write anything too specific in here, as it will probably change / get fixed at some point.)